scripts/docker/alpine.3.6/Dockerfile

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM microsoft/dotnet-buildtools-prereqs:alpine-3.6-3148f11-20171119021156
+
+RUN apk add --no-cache sudo ncurses
```

Comment:
What's `ncurses` for?

Reply:
I don't actually remember, but it was needed for the build to work. I can remove it locally and try what was it if you are interested.

Reply:
So, without ncurses, I get this error:
`/opt/code/scripts/obtain/dotnet-install.sh: line 27: tput: command not found`

Reply:
IMO - this is a bug in the `dotnet-install.sh` script.  The only reason we are using `tput` is because we are trying to print colors on the console.  Printing colors on the console shouldn't be required in order to install use `dotnet-install.sh`.

This is going to mean that anyone who either a) wants to build the dotnet/cli repo or b) wants to use `dotnet-install.sh` is going to have to install this utility on their machine

I don't feel this should block your work.  We can get the initial alpine work in by installing `ncurses` on this docker image.  But I think we should log a bug for `dotnet-install.sh` to not require `tput`, and only use it if it is available.

cc @livarcocc @johnbeisner